packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -127,7 +136,7 @@ class _PreferredAppBarSize extends Size {
 /// {@tool dartpad}
 ///
 ///
-/// ** See code in examples/api/lib/material/app_bar/app_bar.1.dart **
+/// ** See code in examples/api/lib/material/app_bar/app_bar.2.dart **
```

Comment:
Verify that this is the correct link, depending on the description you added above.

Reply:
Verified all 3 examples, thank you!